test/test.js

```diff
@@ -35,16 +36,70 @@ import getPrivateKeyFromWif from "../lib/getPrivateKeyFromWif"
 import getSignature from "../lib/getSignature"
 import verifySignature from "../lib/verifySignature"
 import createAndSendTransaction from "../lib/createAndSendTransaction"
+import getInputAddress from "../lib/getInputAddress"
+import generateSegwitAddress from "../lib/generateSegwitAddress"
+import hashRateFormat from "../lib/hashRateFormat"
 
 const MNEMONIC = "refuse brush romance together undo document tortoise life equal trash sun ask"
 const MNEMONIC2 = "balance blanket camp festival party robot social stairs noodle piano copy drastic"
 const PASSWORD = "julianAssange2020"
 
 describe('js-doichain', function () {
     this.timeout(0);
-    describe('basic doichain functions', function () {
+    describe('basic doichain functions', function () {     
+
+        it("generate a address ", async function () {
+
+           let publicExtendedKey = ''
+            if (publicExtendedKey === "")
+              console.error("please add publicExtendedKey");
+           // const address = getAddress(publicExtendedKey);
+           let addressArray = []
+           let derivationPathOut 
+           let derivationPathIn
+           let network =  bitcoin.networks.bitcoin
+         
+           for (let i = 0; i < 1000; i++) {
+             derivationPathOut = "0/" + i 
+             derivationPathIn = "1/" + i            
+
+             const newAddressOut = generateSegwitAddress(
+               publicExtendedKey,
+               derivationPathOut,
+               network
+             );
+
+             const newAddressIn = generateSegwitAddress(
+               publicExtendedKey,
+               derivationPathIn,
+               network
+             );
+
+             if("bc1q7vtcp3gas4k54y5xtmg2dl7dw599s4wdqha78y" == newAddressOut ){
+                 console.log("_test", "bc1q7vtcp3gas4k54y5xtmg2dl7dw599s4wdqha78y")
+             }
+
+             addressArray[i] = newAddressOut;
+             addressArray[1000 + i] = newAddressIn;
+           }
+           console.log("__addresses", addressArray)
+           //chai.assert(Array.isArray(addressArray), " addressArray is empty"); 
```

Comment:
If you write a test you should never commit a pull request nor a commit without any asserts! A Test without asserts isn't a test.